test/KBaseReport_server_test.py

```diff
@@ -58,12 +60,18 @@ def setUpClass(cls):
         shutil.copy2(os.path.join(dirname, 'data/a.txt'), cls.a_file_path)
         shutil.copy2(os.path.join(dirname, 'data/b.txt'), cls.b_file_path)
         # Upload files to shock
-        cls.a_file_shock = cls.dfu.file_to_shock({
-            'file_path': cls.a_file_path, 'make_handle': 0
-        })
-        cls.b_file_shock = cls.dfu.file_to_shock({
-            'file_path': cls.b_file_path, 'make_handle': 0
-        })
+        try:
+            cls.a_file_shock = cls.dfu.file_to_shock({
+                'file_path': cls.a_file_path, 'make_handle': 0
+            })
+        except Exception as e:
+            print('Unable to store ' + cls.a_file_path + str(e))
+        try:
+            cls.b_file_shock = cls.dfu.file_to_shock({
+                'file_path': cls.b_file_path, 'make_handle': 0
+            })
+        except Exception as e:
+            print('Unable to store ' + cls.b_file_path + str(e))
```

Comment:
There are a load of duplicated commands that could be replaced with a `for` loop that uses `a` and `b`. Could you do a bit of refactoring here to get rid of the duplication?